mdal/frmts/mdal_esri_tin.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2019 Vincent Cloarec (vcloarec at reos dot site)
+*/
+
+#include "mdal_esri_tin.hpp"
+
+//from https://stackoverflow.com/questions/41012414/convert-double-value-from-little-endian-to-big-endian
+template <typename T>
```

Comment:
I think we have a function for this in https://github.com/lutraconsulting/MDAL/blob/master/mdal/frmts/mdal_selafin.cpp#L145

maybe we can just move that function to MDAL utils and use it here too?

Reply:
I have added new function in MDAL utils, and I have changed selafin driver and esri tin driver.
Tell me what do you think about ?
But I am not sure that it is a good idea because the number of bytes read in the file depend now of the size of type. Is there a risk that this size of type is different from platform to other ? If yes, it is maybe not a good idea ...